assets/js/ajax-core.js

```diff
@@ -174,7 +174,9 @@ jQuery(function($){
                 });
 
 				//Check if any cf2 fields are blocking submit
-				var cf2 = window.cf2[ $form.attr( 'id' ) ];
+                if( 'object' === typeof  window.cf2 ){
```

Comment:
@New0 Nitpick, I am about to approve this, but keep in mind arrays, sets and maps are all objects. So `let cf = []; typeof cf` would return "object". In this case, if its an array, then that's intentional misuse, we can't worry about.